<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaemoPackageCreationWidget</class>
 <widget class="QWidget" name="MaemoPackageCreationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>325</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QCheckBox" name="skipCheckBox">
     <property name="toolTip">
      <string>Check this if you want the files below to be deployed directly.</string>
     </property>
     <property name="text">
      <string>Skip packaging step</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Version number:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>13</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Major:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="major">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Minor:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="minor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Patch:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="patch">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="contentsLabel">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Files to deploy:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QTableView" name="packageContentsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="addFileButton">
         <property name="toolTip">
          <string>Add File to Package</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../coreplugin/core.qrc">
           <normaloff>:/core/images/plus.png</normaloff>:/core/images/plus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="removeFileButton">
         <property name="toolTip">
          <string>Remove File from Package</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../coreplugin/core.qrc">
           <normaloff>:/core/images/minus.png</normaloff>:/core/images/minus.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../coreplugin/core.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MaemoPackageCreationWidget</receiver>
   <slot>addFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>732</x>
     <y>525</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>removeFileButton</sender>
   <signal>clicked()</signal>
   <receiver>MaemoPackageCreationWidget</receiver>
   <slot>removeFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>458</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>735</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>skipCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>MaemoPackageCreationWidget</receiver>
   <slot>handleSkipButtonToggled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>major</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MaemoPackageCreationWidget</receiver>
   <slot>versionInfoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>461</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minor</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MaemoPackageCreationWidget</receiver>
   <slot>versionInfoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>patch</sender>
   <signal>valueChanged(int)</signal>
   <receiver>MaemoPackageCreationWidget</receiver>
   <slot>versionInfoChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>466</x>
     <y>-7</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addFile()</slot>
  <slot>removeFile()</slot>
  <slot>handleSkipButtonToggled(bool)</slot>
  <slot>versionInfoChanged()</slot>
 </slots>
</ui>
